.changes/unreleased/Features-20220715-035555.yaml

```diff
@@ -0,0 +1,8 @@
+kind: Features
+body: Support retrying Postgres (and Redshift!) connection attempts in the prescence
```

Comment:
nit: better changelog entry since this change is larger in scope than this body suggests. Maybe something like "Add reusable function for retrying adapter connections. Adds connection retries for Postgres (and Redshift)."

Reply:
Updated :+1:. Agree that the previous message didn't capture the scope.